examples/plot_callosal_tract_profile.py

```diff
@@ -0,0 +1,558 @@
+#
+# The templates are first resampled into the MNI space, before
+# they are brought into the subject's individual native space.
+
+
+print("Fetching callosum ROI templates...")
+
+callosal_templates = afd.read_callosum_templates(resample_to=MNI_T2_img)
+
+show_anatomical_slices(callosal_templates["L_AntFrontal"].get_fdata(),
+                       'Left anterior frontal ROI')
+show_anatomical_slices(callosal_templates["R_AntFrontal"].get_fdata(),
+                       'Right anterior frontal ROI')
+show_anatomical_slices(callosal_templates["Callosum_midsag"].get_fdata(),
+                       'Midsagittal ROI')
+
+print("Creating callosal bundle specification...")
+
+# bundle dict
+bundles = {}
+
+# anterior frontal ROIs
+bundles["AntFrontal"] = {
+    'ROIs': [callosal_templates["L_AntFrontal"],
+             callosal_templates["R_AntFrontal"],
+             callosal_templates["Callosum_midsag"]],
+    'rules': [True, True, True],
+    'cross_midline': True
+}
+
+##########################################################################
+# Tracking:
+# ---------
+# Streamlines are generate using DTI and a deterministic tractography
+# algorithm. For speed, we seed only within the waypoint ROIs for each bundle.
+#
+# .. note::
+#   By default tractography:
+#
+#   - Will identify streamlines with lengths between 10 mm and 1 m, with
+#     turning angles of less than 30.
+#
+#   - Is seeded with a single seed in each voxel on each dimension
+#
+#   - Each step is 0.5 mm
+#
+# .. note::
+#   In this example tractography results in a large number of candidate
+#   streamlines for the anterior forceps.
+
+
+print("Tracking...")
+
+if not op.exists(op.join(working_dir, 'dti_streamlines.trk')):
+    seed_roi = np.zeros(img.shape[:-1])
+    for bundle in bundles:
+        for idx, roi in enumerate(bundles[bundle]['ROIs']):
+            if bundles[bundle]['rules'][idx]:
+                warped_roi = patch_up_roi(
+                    mapping.transform_inverse(
+                        roi.get_fdata().astype(np.float32),
+                        interpolation='linear'),
+                    bundle_name=bundle)
+
+                nib.save(nib.Nifti1Image(warped_roi.astype(float), img.affine),
+                         op.join(working_dir, f"{bundle}_{idx+1}.nii.gz"))
+
+                warped_roi_img = nib.load(op.join(working_dir,
+                                                  f"{bundle}_{idx+1}.nii.gz"))
+                show_anatomical_slices(warped_roi_img.get_fdata(),
+                                       f'warped {bundle}_{idx+1} ROI')
+
+                # Add voxels that aren't there yet:
+                seed_roi = np.logical_or(seed_roi, warped_roi)
+
+    seed_roi_img = nib.Nifti1Image(seed_roi.astype(float), img.affine)
+    nib.save(seed_roi_img, op.join(working_dir, 'seed_roi.nii.gz'))
+
+    show_anatomical_slices(seed_roi_img.get_fdata(), 'Seed ROI')
+
+    tractogram = aft.track(dti_params['params'], seed_mask=seed_roi,
```

Comment:
I have noticed that the docstring are out of date for default parameters for both track and segmentation (perhaps more wide spread)